libraries/exoplayer/src/main/java/androidx/media3/exoplayer/audio/AudioCapabilities.java

```diff
@@ -354,9 +364,9 @@ private static final class Api29 {
     private Api29() {}
 
     @DoNotInline
-    public static int[] getDirectPlaybackSupportedEncodings() {
+    public static int[] getDirectPlaybackSupportedEncodings(int[] encodings) {
```

Comment:
I saw in the latest revision of this pull request, this method is only called at one place, where we pass the return array from `getAllSurroundEncodingsMaybeSupported`. How about we remove the parameter, and call `getAllSurroundEncodingsMaybeSupported` inside of this method?

Reply:
@tianyif  Done. Please see bc197dbd14499569708b0822bc1c9c016c6eabf3. Thanks